File: assets/samples/reverse.whois/classic/php/reverse.whois.php
<?php

$term = 'wikimedia';
$password = 'your whois api password';
$username = 'your whois api username';

$url ="https://www.whoisxmlapi.com/reverse-whois-api/search.php?term1={$term}"
     ."&username={$username}&password={$password}&mode=preview";

print(file_get_contents($url));

File: assets/samples/reverse.whois/classic/python/reverse.whois.py
try:
    from urllib.request import urlopen
except ImportError:
    from urllib2 import urlopen

term = 'wikimedia'
password = 'your whois api password'
username = 'your whois api username'

url = 'https://www.whoisxmlapi.com/reverse-whois-api/search.php?mode=preview'\
    + '&term1=' + term + '&username=' + username + '&password=' + password

print(urlopen(url).read().decode('utf8'))